lib/coradoc/document/video.rb

```diff
@@ -0,0 +1,26 @@
+module Coradoc
+  module Document
+    class Video
+      attr_reader :id, :title, :src, :options
+
+      def initialize(title, options = {})
+        @title = title
+        @id = options.fetch(:id, nil)
+        @src = options.fetch(:src, '')
+        @options = options.fetch(:options, [])
+      end
+
+      def to_adoc
+        anchor = @id ? "[[#{@id}]]\n" : ""
+        title = ".#{@title}\n" unless @title.empty?
+
+        opts = ""
+        if @options.any?
+          opts = %{options="#{@options.join(',')}"}
+        end
+
+        [anchor, title, "video::", @src, "[", opts, "]"].join("")
```

Comment:
I think we need to define the options.

Options need to be validated, e.g. height/width can be `auto` or a pixel number. When given unknown options we should reject.

Reply:
Moved to #42.